spotlight-client/src/SystemNarrativePage/SystemNarrativePage.tsx

```diff
@@ -148,14 +168,20 @@ const SystemNarrativePage: React.FC<{
 
   return (
     <Container>
-      <SectionNavigation
-        activeSection={activeSection}
-        // pagination UI should not respect the scrolling flag;
-        // in fact it should override it, otherwise the buttons
-        // will stop working while the animation is in progress
-        setActiveSection={directlySetActiveSection}
-        totalPages={narrative.sections.length + 1}
-      />
+      <NavContainer>
+        <Sticker>
+          <NavStickyContainer>
+            <SectionNavigation
```

Comment:
I found that clicking on the arrows in the SectionNavigation to click through to the next section to be quite slow. Enough to make me wonder the first time I clicked if something was broken. Have you noticed this?

Reply:
Another question on the section navigation. I was a little frustrated when at the bottom of the page, i couldn't remember which narrative I was on previously so I clicked the 'Back' button, but had to hit back 8 times to actually get back to the previous Narrative. What is the reason to keep the section number in the URL? Are the sections going to be that large that the user will want to navigate between them rather than scrolling between them?

Reply:
hm. it's definitely less snappy in IE 11 but for me it's not quite _that_ bad. There is a delay of about a second or so though. I am not sure what our IE support level is going to be for this product but at the moment I am just aiming for "not completely broken" and by that standard I think this squeaks by? 

Interesting point though about the URLs. I think I tend to agree with you, but this came down as a product requirement, to be able to link directly to a section. We may find that in practice we don't like the way it works. Most of them are not going to be particularly large; I think the majority will fit on a single screen.

I could try replacing the current history item when the section changes, rather than pushing it onto the stack? I don't know if that would just be confusing in a different way, though.

Reply:
I did give the replacement approach a quick try and I feel like it's ....  not obviously better? you wind up on the last section of the previous page if you click on one of those footer links and then hit the back button. It feels ... also weird.  Anything more sophisticated than that, or a change in the URL scheme, is a product discussion I think. I will make a note of it for later review though